### PR TITLE
Collect all python files from current directory

### PR DESCRIPTION
**Pull request by dbasunag**
Currently utils.py are getting skipped.

### CODE REVIEW

**Comment by redhat-qe-bot2** [comment]
Report bugs in [Issues](https://github.com/myakove/github-webhook-server/issues)

The following are automatically added:
 * Add reviewers from OWNER file (in the root of the repository) under reviewers section.
 * Set PR size label.
 * New issue is created for the PR. (Closed when PR is merged/closed)
 * Run [pre-commit](https://pre-commit.ci/) if `.pre-commit-config.yaml` exists in the repo.

Available user actions:
 * To mark PR as WIP comment `/wip` to the PR, To remove it from the PR comment `/wip cancel` to the PR.
 * To block merging of PR comment `/hold`, To un-block merging of PR comment `/hold cancel`.
 * To mark PR as verified comment `/verified` to the PR, to un-verify comment `/verified cancel` to the PR.
        verified label removed on each new commit push.
 * To cherry pick a merged PR comment `/cherry-pick <target branch to cherry-pick to>` in the PR.
    * Multiple target branches can be cherry-picked, separated by spaces. (`/cherry-pick branch1 branch2`)
    * Cherry-pick will be started when PR is merged
 * To build and push container image command `/build-and-push-container` in the PR (tag will be the PR number).
    * You can add extra args to the Podman build command
        * Example: `/build-and-push-container --build-arg OPENSHIFT_PYTHON_WRAPPER_COMMIT=<commit_hash>`
 * To add a label by comment use `/<label name>`, to remove, use `/<label name> cancel`
 * To assign reviewers based on OWNERS file use `/assign-reviewers`
 * To check if PR can be merged use `/check-can-merge`

<details>
<summary>Supported /retest check runs</summary>

 * `/retest tox`: Retest tox
 * `/retest python-module-install`: Retest python-module-install

</details>

<details>
<summary>Supported labels</summary>

 * hold
 * verified
 * wip
 * lgtm

</details>

File: apps/utils.py [hunk 1]
```diff
@@ -82,5 +82,5 @@ def all_python_files() -> Iterable[str]:
             continue
 
         for filename in files:
-            if filename.endswith(".py") and filename != os.path.split(__file__)[-1]:
+            if filename.endswith(".py"):
```
Comment (myakove) [inline apps/utils.py:85]: can you explain why this change is needed?

Comment (dbasunag) [reply apps/utils.py:85]: `os.path.split(__file__)[-1]` is utils.py, so this is basically skipping the utils.py. On the otherhand, this tool is supposed to run from the base directory of a repository, should it still care about current file?

Comment (myakove) [reply apps/utils.py:85]: I think we should collect all Python files, the code that uses this function should exclude files if needed.
This is a generic helper function not tied to a specific operation.

Comment (dbasunag) [reply apps/utils.py:85]: +1. this is why I removed and part
